tests/functional/spec/features/conditional_question_title/conditional_within_repeating_block.spec.js

```diff
@@ -0,0 +1,44 @@
+const helpers = require('../../../helpers');
+
+describe('@watch Feature: Use of conditional Titles in Repeating blocks with condition dependant on answer changing within block', function() {
```

Comment:
Left a watch annotation on this

Reply:
Not quite . On Master the second iteration will invalidate the answer from the first iteration so we would never get to the third . Am changing the test however to explicitly go right through to the confirmation page